src/sentry/api/event_search.py

```diff
@@ -107,15 +106,15 @@ def translate(pat):
 
 event_search_grammar = Grammar(
```

Comment:
Does this apply to issue search as well ? 
That would be a problem because, depending on the filters applied the query may be served entirely by snuba (which would be fine) or by postgres + snuba by doing the join in python.
Queries like 
`assigned_to me OR assigned_to you AND event.message = 'asd'`
won't work because they would turn into 
`(assigned_to me OR assigned_to you) AND event.message = 'asd'`

Reply:
This does not apply to issue search. Using OR/AND in issue search will result in an error.